raphtory/src/db/api/view/graph.rs

```diff
@@ -969,7 +969,10 @@ impl<G: StaticGraphViewOps> SearchableGraphOps for G {
             .get_storage()
             .and_then(|s| s.get_index())
             .ok_or(GraphError::IndexNotCreated)?;
-        index.searcher().search_nodes(self, filter, limit, offset)
+        index
```

Comment:
Just use an empty index and rely on the fallback instead of failing? Index should no longer be required for this now.